Sources/KituraCORS/AllowedOrigins.swift

```diff
@@ -20,7 +20,11 @@ import Foundation
 
 /// Type alias for `AllowedOrigins.regex`.
 #if os(Linux)
+#if swift(>=3.1)
```

Comment:
That was it was called before, and changing it would technically be breaking since it's a public typealias.